snowplow-tracker/build.gradle

```diff
@@ -76,13 +76,16 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:support-annotations:28.0.0'
-    implementation "android.arch.lifecycle:extensions:$project.archLifecycleVersion"
+    implementation 'androidx.annotation:annotation:1.1.0'
+    implementation "androidx.lifecycle:lifecycle-extensions:2.2.0"
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.squareup.okhttp3:okhttp:3.12.12'
+    implementation 'com.squareup.okhttp3:okhttp:4.0.0'
```

Comment:
Same question here

Reply:
I would keep this one in the tracker for now. I'm unsure of possible consequences. I wouldn't set the requirements too high and cut out all the ones with a different version.